### PR TITLE
DCD-386 Vanilla AMIs+Ansible for Bitbucket

### PR DESCRIPTION
**Pull request by tarka**
Blitz-test has been run with the Amsterdam BB support team.

### CODE REVIEW

File: templates/quickstart-bitbucket-dc.template.yaml [hunk 1]
```diff
@@ -628,131 +672,110 @@ Resources:
...
-          packages:
-            !If
-            - SetupCollectd
-            - yum:
-                collectd: []
-                collectd-java: []
-                collectd-generic-jmx: []
-                collectd-rrdtool: []
-            - !Ref AWS::NoValue
+        # configSets:
```
Comment (nanux) [inline templates/quickstart-bitbucket-dc.template.yaml]: Any reasons why you've left commented sections?

File: templates/quickstart-bitbucket-dc.template.yaml [hunk 2]
```diff
@@ -628,131 +672,110 @@ Resources:
...
+            080_run_atl_init_node:
+              command: !Sub |
+                cd /opt/atlassian/dc-deployments-automation/ && ./bin/install-ansible && ./bin/ansible-with-atl-env inv/aws_node_local ${DeploymentAutomationPlaybook} /var/log/ansible-bootstrap.log
+              ignoreErrors: true
+
     Properties:
       AssociatePublicIpAddress: false
       BlockDeviceMappings:
-        - DeviceName: /dev/xvdf
+        - DeviceName: /dev/sdf
```
Comment (nanux) [inline templates/quickstart-bitbucket-dc.template.yaml:764]: Is this due to Amazon Linux 2? Is this also affected by the NVMe disks with different mapping? https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/nvme-ebs-volumes.html

File: templates/quickstart-bitbucket-dc.template.yaml [hunk 3]
```diff
@@ -488,38 +512,58 @@ Conditions:
     !And [!Not [!Equals [!Ref DBMasterUserPassword, '']], Condition: NotStandbyMode]
   UsePublicIp:
     !Equals [!Ref AssociatePublicIpAddress, 'true']
+
 Mappings:
   AWSRegionArch2AMI:
     ap-northeast-1:
-      HVM64: ami-07ff8945520f92958
+      HVM64: ami-0c3fd0f5d33134a76
```
Comment (nanux) [inline templates/quickstart-bitbucket-dc.template.yaml:519]: While doing this, should we consider moving the template to the autoupdated AMIs? https://aws.amazon.com/blogs/compute/query-for-the-latest-amazon-linux-ami-ids-using-aws-systems-manager-parameter-store/

File: templates/quickstart-bitbucket-dc.template.yaml [hunk 4]
```diff
@@ -55,11 +54,12 @@ Metadata:
...
           - HomeVolumeSnapshotId
           - HomeDeleteOnTermination
-          - StartCollectd
+          - DeploymentAutomationRepository
+          - DeploymentAutomationBranch
+          - DeploymentAutomationPlaybook
+          - DeploymentAutomationKeyName
 
     ParameterLabels:
-      AMIOpts:
```
Comment (nanux) [inline templates/quickstart-bitbucket-dc.template.yaml:61]: Do we know how the `AMIOpts` were used before? Was it only for Server AMI?

**Review by nanux — APPROVED** [review]

File: templates/quickstart-bitbucket-dc.template.yaml (hunk 1 above)
Comment (tarka) [reply templates/quickstart-bitbucket-dc.template.yaml]: Missed them during clean-up; removed.

File: templates/quickstart-bitbucket-dc.template.yaml (hunk 3 above)
Comment (tarka) [reply templates/quickstart-bitbucket-dc.template.yaml:519]: Definitely. I just didn't want to bundle additional changes at this point, so I'd suggest doing it for all templates as a separate change.

File: templates/quickstart-bitbucket-dc.template.yaml (hunk 4 above)
Comment (tarka) [reply templates/quickstart-bitbucket-dc.template.yaml:61]: No ideal. It's not used in the old AMIs, just a passing reference in some test code.

File: templates/quickstart-bitbucket-dc.template.yaml (hunk 2 above)
Comment (tarka) [reply templates/quickstart-bitbucket-dc.template.yaml:764]: I tried using xvdf and had some issues, so just used sdf.

**Pull request merged**